app/controllers/api/v1/player_points_controller.rb

```diff
@@ -1,5 +1,4 @@
 class Api::V1::PlayerPointsController < ApplicationController
-  skip_before_action :authorize!
```

Comment:
@katiekeel 

With this stripped out, does devise take care of authorization under the hood?

Reply:
Not really. Current_user is still the same as it was. So we can still use the same checks we wrote before that uses it. I fixed the authorization tests that failed so we should be ok 👍

Reply:
Sounds good. Merging.

🌮